config/routes.rb

```diff
@@ -169,8 +171,14 @@
     resources :schools, only: %i[index edit update], controller: "publishers/organisations/schools"
   end
 
-  # Legacy publisher sign in path (users may still have this bookmarked)
-  get "/identifications/new", to: redirect("/publishers/sign_in")
+  # Legacy authentication paths (users may still have this bookmarked)
+  get "/identifications/new", to: redirect("/publishers/sign-in")
+  get "/publishers/sign_in", to: redirect("/publishers/sign-in")
+  get "/publishers/sign_out", to: redirect("/publishers/sign-out")
+  get "/publishers/account_requests/new", to: redirect("/publishers/account-requests/new")
+  get "/jobseekers/sign_in", to: redirect("/jobseekers/sign-in")
+  get "/teaching-jobs-for-:job_role", to: redirect { |path_params| "/teaching-jobs-for-#{path_params[:job_role].tr('_', '-')}" },
```

Comment:
It looks like this is under the heading `# Legacy authentication paths (users may still have this bookmarked)` - could you edit that comment to not refer to authentication?